plugins/modules/dns_host.py

```diff
@@ -0,0 +1,482 @@
+                    type: str
+                    returned: Always
+                domain:
+                    description:
+                        - "Kerberos realm of the principal."
+                    type: str
+                    returned: Always
+                key:
+                    description:
+                        - "The resource identifier."
+                    type: str
+                    returned: Always
+                principal:
+                    description:
+                        - "Kerberos principal associated with key."
+                    type: str
+                    returned: Always
+                uploaded_at:
+                    description:
+                        - "Upload time for the key."
+                    type: str
+                    returned: Always
+                version:
+                    description:
+                        - "The version number (KVNO) of the key."
+                    type: int
+                    returned: Always
+        name:
+            description:
+                - "Host display name."
+            type: str
+            returned: Always
+        ophid:
+            description:
+                - "On-Prem Host ID."
+            type: str
+            returned: Always
+        protocol_absolute_name:
+            description:
+                - "Host FQDN in punycode."
+            type: str
+            returned: Always
+        provider_id:
+            description:
+                - "External provider identifier."
+            type: str
+            returned: Always
+        server:
+            description:
+                - "The resource identifier."
+            type: str
+            returned: Always
+        site_id:
+            description:
+                - "Host site ID."
+            type: str
+            returned: Always
+        tags:
+            description:
+                - "Host tagging specifics."
+            type: dict
+            returned: Always
+        type:
+            description:
+                - "Defines the type of host. Allowed values:"
+                - "* I(bloxone_ddi): host type is BloxOne DDI,"
+                - "* I(microsoft_azure): host type is Microsoft Azure,"
+                - "* I(amazon_web_service): host type is Amazon Web Services,"
+                - "* I(microsoft_active_directory): host type is Microsoft Active Directory,"
+                - "* I(google_cloud_platform): host type is Google Cloud Platform."
+            type: str
+            returned: Always
+"""  # noqa: E501
+
+from ansible_collections.infoblox.bloxone.plugins.module_utils.modules import BloxoneAnsibleModule
+
+try:
+    from dns_config import Host, HostApi
+    from universal_ddi_client import ApiException, NotFoundException
+except ImportError:
+    pass  # Handled by BloxoneAnsibleModule
+
+
+class HostModule(BloxoneAnsibleModule):
```

Comment:
```suggestion
class DnsHostModule(BloxoneAnsibleModule):
```

We have Infra Host and Ipam Host too , cant use generic `HostModule` name

Reply:
done